### ISSUE TITLE
Fixed font not set in QFontDatabase

### ISSUE THREAD

**Issue by omaemae** [cyberos/cyber-settings#14]
At least I assume that's what's happening.
![image](https://user-images.githubusercontent.com/17727163/102562312-19cddc00-40d7-11eb-82ef-9ca367900115.png)

@rekols

**Comment by reionwong** [comment]
Finally to solve this problem, I am developing a file manager :)

**Comment by itsvic-dev** [comment]
Yeah, I noticed you started rewriting the file manager.

**Comment by itsvic-dev** [comment]
Yup, my suspicions are confirmed, and it occurs for the standard system font as well.
![image](https://user-images.githubusercontent.com/17727163/102563921-ae860900-40da-11eb-9c2b-0396738470a6.png)

**Comment by itsvic-dev** [comment]
Seems to be primarily a settings daemon issue.
![image](https://user-images.githubusercontent.com/17727163/102564161-22281600-40db-11eb-8ec6-9a237a6f3fb4.png)
